CHANGELOG.md

```diff
@@ -14,6 +14,7 @@
 
 💅 *Improvements*
 * `sdk.current_run_ids()` now returns a `NamedTuple` called `CurrentRunIDs` to help with typing.
+* Task that requests ray-incompatible number of resources throws an error at submission time
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**question:** ‏What about workflow resources?

Reply:
Honestly - im not super confident on how those are parsed. Workflow resources are not things we pass to ray directly. Im gonna start slack thread if there are any limitations on workflow resources